### PR TITLE
Robot state port

### PR DESCRIPTION
**Pull request by ahcorde**
@henningkayser  I have applied all the changes that you requested in a this new PR. Please close this one https://github.com/ros-planning/moveit2/pull/59

### CODE REVIEW

**Comment by ahcorde** [comment]
@davetcoleman fixed CMakelists.txt issues

File: moveit_core/robot_state/src/conversions.cpp [hunk 1]
```diff
@@ -530,8 +537,8 @@ void streamToRobotState(RobotState& state, const std::string& line, const std::s
   {
     // Get a variable
     if (!std::getline(line_stream, cell, separator[0]))
-      ROS_ERROR_STREAM_NAMED(LOGNAME, "Missing variable " << state.getVariableNames()[i]);
-
+      // ROS_ERROR_STREAM_NAMED(LOGNAME.c_str(), "Missing variable " << state.getVariableNames()[i]);
```
Comment (henningkayser) [inline moveit_core/robot_state/src/conversions.cpp]:
```suggestion
```

File: moveit_core/robot_state/test/robot_state_benchmark.cpp [hunk 2]
```diff
@@ -38,8 +38,8 @@
 #include <moveit/robot_state/robot_state.h>
 #include <moveit/utils/robot_model_test_utils.h>
 #include <eigen_stl_containers/eigen_stl_containers.h>
-#include <gtest/gtest.h>
 #include <chrono>
+#include <gtest/gtest.h>
```
Comment (henningkayser) [inline moveit_core/robot_state/test/robot_state_benchmark.cpp]: How are the changes in this file related to the port?

Comment (ahcorde) [reply moveit_core/robot_state/test/robot_state_benchmark.cpp]: it's not part of the port (code-wise) but I had to modify the CMakelists.txt with `ament_add_gtest`. So I used this opportunity to also fix the tests.

**Comment by mlautman** [comment]
Great!

**Pull request merged**